app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: int
```

Comment:
The `brand` attribute is currently an integer, but it might be more appropriate to use a string to represent the brand name of the car.